programs/server/Server.cpp

```diff
@@ -680,6 +679,7 @@ int Server::main(const std::vector<std::string> & /*args*/)
     Poco::ThreadPool server_pool(3, config().getUInt("max_connections", 1024));
     std::mutex servers_lock;
     std::vector<ProtocolServerAdapter> servers;
+    std::vector<ProtocolServerAdapter> updated_servers;
```

Comment:
it doesn't seem we have any reason to split the vector of servers and thus to make the logic more complex (I mean calculation of current connections, waiting, metrics).

Reply:
The purpose of keeping it separately was to distinguish servers that are in progress of stopping and currently active, yes you can distinguish it by `isStopping`, but serparate list of servers looks more natural.

But I agree, that it is possible to forget list of this servers somewhere, so OK, I will reverted this part too.